references/cli/status-ui/model/cluster.go

```diff
@@ -0,0 +1,85 @@
+package model
+
+import (
+	"context"
+	"strings"
+
+	"github.com/oam-dev/cluster-gateway/pkg/config"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	prismclusterv1alpha1 "github.com/kubevela/prism/pkg/apis/cluster/v1alpha1"
+)
+
+type Cluster struct {
+	name        string
+	alias       string
+	clusterType string
+	endpoint    string
+	labels      string
+}
+
+type ClusterList struct {
+	title []string
+	data  []Cluster
+}
+
+func (l *ClusterList) Header() []string {
+	return l.title
+}
+
+func (l *ClusterList) Body() [][]string {
+	data := make([][]string, 0)
+	for _, cluster := range l.data {
+		data = append(data, []string{cluster.name, cluster.alias, cluster.clusterType, cluster.endpoint, cluster.labels})
+	}
+	return data
+}
+
+func ListClusters(ctx context.Context, c client.Client) *ClusterList {
+	list := &ClusterList{
+		title: []string{"Name", "Alias", "Type", "EndPoint", "Labels"},
+	}
+	name, ok := ctx.Value("appName").(string)
+	if !ok {
+		return list
+	}
+	ns, ok := ctx.Value("appNs").(string)
+	if !ok {
+		return list
+	}
+	app, err := LoadApplication(c, name, ns)
+	if err != nil {
+		return list
+	}
+	clusterSet := make(map[string]interface{})
+
+	for _, svc := range app.Status.AppliedResources {
+		if svc.Cluster == "" {
+			clusterSet["local"] = struct{}{}
```

Comment:
There is one const named "ClusterLocalName" in multicluster.go。 It is better to use it , not String "local"

Reply:
ok, I have fixed it.